marklogic-sesame/src/test/java/com/marklogic/semantics/sesame/MarkLogicRepositoryConnectionTest.java

```diff
@@ -1224,7 +1224,7 @@ public void testNestedConnections()
         config.setPort(port);
         config.setUser(user);
         config.setPassword(password);
-        config.setAuth("DIGEST");
+        config.setAuth("BASIC");
```

Comment:
The reason for this switch was just my test infrastructure. I had a server that ran on BASIC Auth and thats why I made that switch.